testing/jupyterhub_config.py

```diff
@@ -4,7 +4,7 @@
 to enable testing without administrative privileges.
 """
 
-c = get_config()  # noqa
+c = get_config()
```

Comment:
> This should keep # noqa for the undefined name on get_config()

It is a manual commit where I remove it, after concluding that we have `get_config()` as an excluded thing in config.

Do you think we should retain it because people may copy-paste it and not have our flake8 config for example, or that it could help people get its something magic about it?

Reply:
Decision: remove or retain ade8999?

Reply:
https://github.com/jupyterhub/jupyterhub/blob/5ea93add289f9905e4957f8a1a8d6d01ce5ab634/.flake8#L12

Reply:
Yes, that's precisely why - we exclude it, but folks who _use_ the examples won't carry over our config. (we could maybe even remove our `builtins` flake8 config to ensure we add these, but no need in this PR).

traitlets generated config files now do the same thing: https://github.com/ipython/traitlets/pull/750

Reply:
:+1: sorry for the noise! Rebased and didn't pick that commit now.